trunk/SUAVE/Methods/Weights/Correlations/BWB/systems.py

```diff
@@ -0,0 +1,130 @@
+# systems.py
```

Comment:
I think it would be a good idea for clarity. In this correlation I have used some of the tubular and wing correlations in addition to some that you provided me with (" aft_centerbody.py, cabin.py ). Let me know your thoughts.